src/shmem_internal.h

```diff
@@ -337,7 +337,7 @@ void* shmem_internal_get_next(intptr_t incr);
 static inline double shmem_internal_wtime(void) {
     double wtime = 0.0;
 
-#ifdef CLOCK_MONOTONIC
+#ifdef HAVE_CLOCK_GETTIME
```

Comment:
I don't think it's necessary to check.  This is one of the standard clocks, so it should be supported by any modern Linux platform (or another way to look at this is that if you get a build failure here, there is probably an issue with your platform and it's better to raise an error than to silently give a low resolution timer).  We were using ```CLOCK_MONOTONIC``` as a gate in the old code, because it was the easiest way to check if the function exists.  The new configure check does two things -- it looks for ```clock_gettime``` and if it finds it, it defines ```HAVE_CLOCK_GETTIME``` and it also adds ```librt``` to the linker flags.